lib/x11utils.pm

```diff
@@ -121,6 +131,7 @@ sub ensure_unlocked_desktop {
         if (match_has_tag('screenlock') || match_has_tag('blackscreen')) {
             wait_screen_change {
                 send_key 'esc';                                                                           # end screenlock
+                record_info("Screen lock present");
```

Comment:
IMHO an overuse of `record_info`. Common operations like matching on a screen or pressing a key should not be accompanied with a `record_info`.